src/Mailjet/Client.php

```diff
@@ -87,8 +91,8 @@ private function _call($method, $resource, $action, $args)
      */
     private function getApiUrl()
     {
-        $h = $this->secure ? 'https' : 'http';
-        return $h . '://api.mailjet.com/' . $this->version;
+        $h = $this->secured === true ? 'https' : 'http';
```

Comment:
No advantage for the strict checking or the normal one. I'm just used to do it because I don't expect any other kind of variable than a boolean.